### PR TITLE
[FIX] hr: remove deadcode of KanbanView/openChat

### PR DESCRIPTION
**Pull request by alexkuhn**
OpenChat feature in `hr` is managed by the view widget "hr_employee_chat", which is added in kanban template.
The override of Kanban View to add openChat feature is not called at all, so this commit removes this dead code, and the js_class, as its only purpose was adding openChat.

https://github.com/odoo/enterprise/pull/42897

### CODE REVIEW

File: addons/hr/static/src/views/kanban_view.js [hunk 1]
```diff
@@ -5,15 +5,11 @@ import { registry } from '@web/core/registry';
...
-    async openChat(employeeId) {
-        this.model.env.services["mail.messaging"].openChat({ employeeId });
-    }
-}
+export class EmployeeKanbanRecord extends KanbanModel.Record {}
 
 export class EmployeeKanbanModel extends KanbanModel {}
 
-EmployeeKanbanModel.services = [...KanbanModel.services, "mail.messaging"];
+EmployeeKanbanModel.services = [...KanbanModel.services];
```
Comment (tsm-odoo) [inline addons/hr/static/src/views/kanban_view.js:12]: Should remove custom kanban + `js_class`

Comment (alexkuhn) [reply addons/hr/static/src/views/kanban_view.js]: done!

**Comment by tsm-odoo** [comment]
@robodoo r+

**Comment by alexkuhn** [comment]
@robodoo r+

**Comment by alexkuhn** [comment]
@robodoo r+

**Pull request closed without merging**